RetrospectiveExtension.Frontend/components/feedbackBoardContainer.tsx

```diff
@@ -718,8 +722,8 @@ export default class FeedbackBoardContainer extends React.Component<FeedbackBoar
   }
 
   private createBoard = async (title: string, maxvotesPerUser: number, columns: IFeedbackColumn[], isIncludeTeamEffectivenessMeasurement: boolean, isBoardAnonymous: boolean, shouldShowFeedbackAfterCollect: boolean, displayPrimeDirective: boolean) => {
-const createdBoard = await BoardDataService.createBoardForTeam(this.state.currentTeam.id, title, maxvotesPerUser, columns, isIncludeTeamEffectivenessMeasurement, isBoardAnonymous, shouldShowFeedbackAfterCollect, displayPrimeDirective);
-console.log(createdBoard);
+    const createdBoard = await BoardDataService.createBoardForTeam(this.state.currentTeam.id, title, maxvotesPerUser, columns, isIncludeTeamEffectivenessMeasurement, isBoardAnonymous, shouldShowFeedbackAfterCollect, displayPrimeDirective);
+    console.log(createdBoard);
```

Comment:
did we forget to remove debug related `console.log`? if it's for debugging purposes, I'd like having no `console.log` in the code, if it's not for debugging, we should a sentence that describes what's the log for.

Reply:
It's actually there in the original code. Happy to remove it now that you mention it.